### PR TITLE
Twitter tutorial

### PR DESCRIPTION
**Pull request by emile-00**
Added new Twitter tutorial

## Info
- **Description**: 
[ What's changed? ]

- **Related code PR**: 
[ Provide link to the code PR here. For example, https://github.com/risingwavelabs/risingwave/pull/4085. Delete this part if there's no code PR related. ]

- **Related doc issue**: 
Resolves [ Provide link to the doc issue here. ]

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [x] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview and the updated parts look all good. <details><summary>How?</summary>image</details>
  - [ ] (For version-specific PR) I have confirmed that this PR is for a released version. If the software version is not released, put it on hold.

## Published doc pages
  [ After merging this PR, edit the description to include the links to the updated doc pages here. For example, https://www.risingwave.dev/docs/latest/intro/. ]

### CODE REVIEW

File: docs/tutorials/fast-twitter-events-processing.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+```shell
+docker-compose down -v
+```
+
+## Summary
+
+In this tutorial, we learn:
+
+* How to define a nested table with RisingWave
+* How to extract character combinations from a string using RegEx
```
Comment (hengm3467) [inline docs/tutorials/fast-twitter-events-processing.md]: @emile-00 "RegEx" is not a common abbreviation. I suggest we spell out "regular expressions".

**Review by hengm3467 — COMMENTED** [review]
Added one comment about "RegEx". Otherwise looks good to me.

**Review by hengm3467 — APPROVED** [review]
Looks good to me.

File: docs/tutorials/fast-twitter-events-processing.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+In the demo cluster, we packaged RisingWave and a workload generator. The workload generator will start to generate random traffic and feed them into Kafka as soon as the cluster is started.
+
+First, clone the [risingwave-demo](https://github.com/singularity-data/risingwave-demo) repository to the environment.
+
+```shell
+git clone https://github.com/singularity-data/risingwave-demo.git
+```
+
+Now navigate to the `ad-ctr` directory and start the demo cluster from the docker compose file.
```
Comment (neverchanje) [inline docs/tutorials/fast-twitter-events-processing.md]:
```suggestion
Now navigate to the `twitter` directory and start the demo cluster from the docker compose file. 
```

File: docs/tutorials/fast-twitter-events-processing.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+The `regexp_matches` function will find all the text in the tweet that matches the RegEx pattern `#\w+`. This extracts all the hashtags from the tweet and stores them in an array.
+
+```
+       hashtag	      |     created_at
+----------------------+--------------------
+[#RisingWave, #cloud] | 2022-05-18 17:00:00
+```
+
+Then the `unnest` function will separate each item in the array into separate rows.
+```
```
Comment (neverchanje) [inline docs/tutorials/fast-twitter-events-processing.md:107]: I was gonna use markdown table syntax

**Review by neverchanje — APPROVED** [review]

**Pull request merged**